coordinator/main.go

```diff
@@ -103,20 +112,30 @@ func run() (retErr error) {
 			return fmt.Errorf("failed to listen: %w", err)
 		}
 		if err := grpcServer.Serve(lis); err != nil {
+			logger.Error("serving Coordinator API", "err", err)
```

Comment:
```suggestion
			logger.Error("serving coordinator API", "err", err)
```

Reply:
Coordinator is a proper noun and always written capitalized.